src/components/Shared/Message.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from '@components/UI/Button';
+import { Profile } from '@generated/types';
+import { Client } from '@xmtp/xmtp-js';
+import { FC } from 'react';
+import toast from 'react-hot-toast';
+import { SIGN_WALLET } from 'src/constants';
+import { useAppStore } from 'src/store/app';
+import { useSigner } from 'wagmi';
+
+interface Props {
+  profile: Profile;
+}
+
+const Message: FC<Props> = ({ profile }) => {
+  const currentProfile = useAppStore((state) => state.currentProfile);
+  const { data: signer, isError, isLoading } = useSigner();
+
+  async function sendGm(): Promise<void> {
```

Comment:
**suggestion:** can we make it as arrow function 🙏🏼

Reply:
done!